tests/Integration/inc/common/doAdminPostRocketPurgeCache.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\Inc\Common;
+
+use Brain\Monkey\Functions;
+use WPMedia\PHPUnit\Integration\TestCase;
+
+/**
+ * @covers ::do_admin_post_rocket_purge_cache
+ * @group admin
+ * @group Common
+ * @group AdminOnly
+ */
+class Test_DoAdminPostRocketPurgeCache extends TestCase {
+
+	public function setUp() {
+		parent::setUp();
+
+		$admin = get_role( 'administrator' );
+		$admin->add_cap( 'rocket_purge_cache' );
+		$user = $this->factory->user->create( [ 'role' => 'administrator' ] );
+		wp_set_current_user( $user );
+
+		Functions\when( 'wp_safe_redirect' )->justReturn( null );
+		Functions\when( 'wp_die' )->justReturn( null );
+
+		require_once WP_ROCKET_PLUGIN_ROOT . 'inc/common/purge.php';
```

Comment:
Is that needed? The file should already be loaded by the plugin at that point

Reply:
Fixed in f873828.